Mathlib/RingTheory/Radical.lean

```diff
@@ -38,6 +38,8 @@ open scoped Classical
 
 open UniqueFactorizationMonoid
```

Comment:
```suggestion
```

Reply:
Without it I can't use `normalizedFactors`, isn't it? I just checked locally.

Reply:
Oh you're putting everything in the `UniqueFactorizationDomain` namespace, not `UniqueFactorizationMonoid`! I strongly suggest you stick to `UniqueFactorizationMonoid` instead of creating yet another namespace

Reply:
Updated!